prow/github/client.go

```diff
@@ -1002,3 +1039,29 @@ func (c *Client) Merge(org, repo string, pr int, details MergeDetails) error {
 
 	return nil
 }
+
+// ListCollaborators gets a list of all users who have access to a repo (and can become assignees
+// or requested reviewers). This includes, org members with access, outside collaborators, and org
+// owners.
+func (c *Client) ListCollaborators(org, repo string) ([]User, error) {
+	c.log("ListCollaborators", org, repo)
+	if c.fake {
+		return nil, nil
+	}
+	path := fmt.Sprintf("/repos/%s/%s/collaborators", org, repo)
+	var users []User
+	err := c.readPaginatedResults(
+		path,
+		"application/vnd.github.hellcat-preview+json",
```

Comment:
Can you add a comment where you link to a doc or comment about the nature of this accept header?

Reply:
Done.